Analyse_Birds.qmd

```diff
@@ -457,7 +455,268 @@ Here we have **58%** of good predictions, which is somewhat below the common thr
+# Then we check for significance
+drop1(mod1,test="Chi")
+```
+We decide to remove the less significative variable : here it's Year.
+
+```{r poisson_model1,include=TRUE}
+# Model formulation
+mod1<-glm(Number_Fledged~ County
+        + Species
+        + Habitat
+        ,data=data_filtered
+        ,family=poisson(link="log"))
+#type de la variable : poisson et fonction de lien de type log
+# Then we check for significance
+drop1(mod1,test="Chi")
+```
+
+Now the less significative is Habitat
+
+```{r poisson_model2,include=TRUE}
+# Model formulation
+mod1<-glm(Number_Fledged~ County
+        + Species
+        ,data=data_filtered
+        ,family=poisson(link="log"))
+#type de la variable : poisson et fonction de lien de type log
+# Then we check for significance
+drop1(mod1,test="Chi")
+```
+
+County is still not significative so we have to remove it.
+
+```{r poisson_model3,include=TRUE}
+# Model formulation
+mod1<-glm(Number_Fledged~ Species
+        ,data=data_filtered
+        ,family=poisson(link="log"))
+#type de la variable : poisson et fonction de lien de type log
+# Then we check for significance
+drop1(mod1,test="Chi")
+
+summary(mod1)
+```
+
+We have the final model with only the Species.
+
+-   $Species{Bobolink }$ = 0 (intercept)\
+-   $Species{Dickcissel }$ = $-0.18^{**}$
+-   $Species{Eastern Meadowlark }$ = $-0.14^{**}$
+-   $Species{Grasshopper Sparrow }$ = $-0.05$
+-   $Species{Henslow's Sparrow }$ = $-0.18$
+-   $Species{Savannah Sparrow }$ = $-0.19^{***}$
+-   $Species{Vesper Sparrow }$ = $-0.32^{***}$
+-   $Species{Western Meadowlark }$ = $-0.04$
+
+The equation of the model is the following:
+
+$$ Number_Fledged = 1.29^{***} + (Species{Bobolink} = 0 ;\:Species{Dickcissel} = -0.18^{**} ; \:Species{Eastern Meadowlark} = -0.14^{**} ; \:Species{Grasshopper Sparrow} = -0.05 ; \:Species{Henslow's Sparrow} = -0.18 ; \:Species{Savannah Sparrow = -0.19^{***} ; \:Species{Vesper Sparrow = -0.32^{***} ; \:Species{Western Meadowlark} = -0.04) $$
+
+According to the model's summary, the null deviance is of 599.76 and residual deviance is of 572.82 in order to estimate the explained deviance, one can use the following formula:\
+
+$$Pseudo\:R^2=100\:.\:\frac{Null\:Deviance- Residual\:Deviance}{Null\:Deviance}$$
+
+```{r poisson_deviance, include=TRUE}
+# Estimate of deviance explained
+(mod1$null.deviance-mod1$deviance)/mod1$null.deviance
+```
+
+This is very very bad... The variable species doesn't seem to explain a
+lot of the variation in the number of young individuals in the nest.
+
+
+## C. Model validation:
+
+The model that we have done is a generalized linear model so there aren't residual normality or homogeneity. However, it is important to check the independence of residuals.
+
+First, when there is counting model like when a Poisson distribution is followed, the overdispersion has to be checked. It can be done with the following code. If the result is over 1, there are overdispersion and the model should be reconsidered.
+
```

Comment:
it is grammaticaly correct but i would maybe say : First of all, in situations where a counting model is used, such as when following a Poisson distribution